docs/en/operations/settings/settings.md

```diff
@@ -4203,6 +4203,62 @@ Default value: `0`.
 Use this setting only for backward compatibility if your use cases depend on old syntax.
 :::
 
+## session_timezone {#session_timezone}
+
+If specified, sets an implicit timezone (instead of [server default](../server-configuration-parameters/settings.md#server_configuration_parameters-timezone).
+All DateTime/DateTime64 values (and/or functions results) that have no explicit timezone specified are treated as having this timezone instead of default.
+A value of `''` (empty string) configures the session timezone to the server default timezone.
```

Comment:
Well, checking if string is empty is cheap, easy and obvious, and it also doesn't break the way things currently work, while empty string is passed by default, e.g. in TimezoneMixin and all its children.
This would introduce additional checks and string comparisons here and there, not good for performance also.
Moreover, in all other settings emptiness means that setting is not set and default behavior is working.
IMO when one wants to change back to default, empty string looks more default than some string with empty value, especially if he has used other settings previously

Reply:
Makes sense, thanks.